packages/dd-trace/src/appsec/index.js

```diff
@@ -6,6 +6,7 @@ const remoteConfig = require('./remote_config')
 const {
   bodyParser,
   cookieParser,
+  expressProcessParams,
```

Comment:
can we try to keep consistency of order between channel.js, appsec/index.js require, subscribe, handler, and unsubscribe ? just so it's not all over the place and it's easier to find a specific one.

Reply:
✅